src/QsCompiler/DataStructures/Diagnostics.fs

```diff
@@ -335,8 +333,6 @@ type WarningCode =
     | SetInResultConditionedBlock = 5026
     | UnsupportedCallableCapability = 5027
 
-    | NamespaceAleadyOpen = 6003
```

Comment:
Interestingly, removing this enum value breaks IQ#, since it uses this value internally. Rather than require this to be a cross-repo change that also updates the uses of this enum there, I would recommend keeping this file unchanged and just removing the uses in this repo.

Reply:
Thanks, I didn't even consider that I was breaking a public API here